lib/fastlane/actions/hipchat.rb

```diff
@@ -32,15 +32,15 @@ def self.run(params)
         if api_version.to_i == 1
           ########## running on V1 ##########
           if user?(channel)
-            raise 'HipChat private message not working with API V1 please use API V2 instead'.red
+            fail 'HipChat private message not working with API V1 please use API V2 instead'.red
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.